composer.json

```diff
@@ -54,6 +56,10 @@
         ],
         "post-update-cmd": [
             "@php artisan vendor:publish --tag=laravel-assets --ansi --force"
+        ],
+        "dev": [
```

Comment:
@jasonvarga Before we merge this, can you think of anything Statamic-specific that might be helpful to have here?